ci-operator/step-registry/upi/conf/libvirt/network/upi-conf-libvirt-network-commands.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# install jq if not installed
+if ! [ -x "$(command -v jq)" ]; then
+    mkdir -p /tmp/bin
+    JQ_CHECKSUM="2f312b9587b1c1eddf3a53f9a0b7d276b9b7b94576c85bda22808ca950569716"
+    curl -Lo /tmp/bin/jq "https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-amd64"
+
+    actual_checksum=$(sha256sum /tmp/bin/jq | cut -d ' ' -f 1)
+    if [ "${actual_checksum}" != "${JQ_CHECKSUM}" ]; then
+        echo "Checksum of downloaded JQ didn't match expected checksum"
+        exit 1
+    fi
+
+    chmod +x /tmp/bin/jq
+    export PATH=${PATH}:/tmp/bin
+fi
```

Comment:
There should be `yq` in the `libvirt-installer` image: https://github.com/openshift/installer/blob/master/images/libvirt/Dockerfile.ci#L31-L36. We even have multi-arch `yq` container images created by @aleskandro  that should be used instead (https://github.com/openshift/installer/blob/master/images/installer/Dockerfile.upi.ci#L16-L17).
If that's not enough, we should consider adding `jq` as well so we don't have to download it in every run.

Reply:
Thanks for the note.
1. I didn't realize that yq can parse json.
2. I didn't realize that Alessandro had already built/published a X-arch image. :)

I'll see if I can use the yq4 image in place here, and if it works, I'll throw in a backlog task to replace jq in all our workflows. :)